.github/workflows/gh-pages.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   build-deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2.3.4
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses
> Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work
🤔

Reply:
I have no idea where to ask why does the Dependabot fix these 😁

Reply:
I found this thread: https://github.community/t/increase-if-necessary-for-github-actions-in-dependabot/179581